src/core/linux/config-loader.hpp

```diff
@@ -48,14 +50,21 @@ class ConfigLoader {
 		 * known working main system configuration.
 		 */
 		if (const char *config_file_path = std::getenv("IPTSD_CONFIG_FILE")) {
+			spdlog::info("Loading config {}, specified with IPTSD_CONFIG_FILE.",
+			             config_file_path);
 			this->load_file(config_file_path);
 			return;
 		}
 
-		if (std::filesystem::exists(common::buildopts::ConfigFile))
+		if (std::filesystem::exists(common::buildopts::ConfigFile)) {
+			spdlog::info("Loading config {}.", common::buildopts::ConfigFile);
 			this->load_file(common::buildopts::ConfigFile);
+		}
 
 		this->load_dir(common::buildopts::ConfigDir, false);
+
+		if (!m_loaded_config)
+			spdlog::warn("No config file loaded at all, this is not good.");
```

Comment:
This is not really an issue, because all config values have defaults, and newer devices don't need presets, so I wouldn't print a warning because of it.

Reply:
It is implicit... which means that the ones that know it is implicit won't encounter the footgun. I wasted 30 minutes before I started the daemon with `strace` to see I placed my config file in the wrong location >_<

Either way, your code, your call; I'll remove it.

Reply:
My point is kinda that if you *want* to `rm -rf` your config then iptsd shouldn't try to prevent you from doing it, because it won't break anything.

How about this? Neutrally printing that nothing was done seems like a good compromise.

```suggestion
			spdlog::info("No config file loaded, using default values.");
```

Reply:
Added back in 1623671b32151f52cefffb0043235bb0ef515813, please resolve this if you're happy with the current form.